server.js

```diff
@@ -47,17 +49,22 @@ app.get('/api', function apiIndex(req, res) {
   // It would be seriously overkill to save any of this to your database.
   // But you should change almost every line of this response.
   res.json({
-    woopsIForgotToDocumentAllMyEndpoints: true, // CHANGE ME ;)
     message: "Welcome to my personal api! Here's what you need to know!",
-    documentationUrl: "https://github.com/example-username/express-personal-api/README.md", // CHANGE ME
-    baseUrl: "http://YOUR-APP-NAME.herokuapp.com", // CHANGE ME
+    documentationUrl: "https://github.com/balbini/express-personal-api",
+    baseUrl: "https://pure-brook-11168.herokuapp.com/",
     endpoints: [
       {method: "GET", path: "/api", description: "Describes all available endpoints"},
-      {method: "GET", path: "/api/profile", description: "Data about me"}, // CHANGE ME
-      {method: "POST", path: "/api/campsites", description: "E.g. Create a new campsite"} // CHANGE ME
+      {method: "GET", path: "/api/profile", description: "Data about Bryan"},
+      {method: "GET", path: "/api/heroes", description: "Index of all heroes"},
+      {method: "POST", path: "/api/heroes", description: "Create a new hero."},
+      {method: "DELETE", path: "/api/heroes/:Id", description:"Remove a hero from the database."}
     ]
   })
 });
+app.get("/api/profile", controllers.profile.index);
+app.get("/api/heroes", controllers.heroController.index);
+app.post("/api/heroes", controllers.heroController.create);
+app.delete("/api/heroes/:Id", controllers.heroController.delete)
```

Comment:
A little weird to use capital letters for your params (although it obviously works)